pilot/pkg/config/clusterregistry/types.go

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
Is there a need for this file? The K8s Cluster registry has been approved for use for Istio. Do you think you could use the following in Gopkg.toml and have your code use it:
required = [
...
"k8s.io/cluster-registry/pkg/apis/clusterregistry/v1alpha1"
]
[[constraint]]
name = "k8s.io/cluster-registry"
revision = "a636176fd5730d0638f1dee29307d6073eb0ba72"

Reply:
I will give it shot as I indicated above.

Reply:
Well, I'm a bit worried about what deps it'll bring in - and please don't use SHA revision.
Either branch or master. SHA is only used as last resort. 

I was thinking a simple json or minimal config file would simplify this - and we can add the cluster registry when we are ready to use it fully. We just need a struct with 5 fields, how many deps and layers does it need ( for file based ) ?

Reply:
@costinm It's done per https://docs.google.com/document/d/1cOxPnTpnVWKR0m7a6PYJ-0NSTZ-e2J1IULnfojz7OZ4/edit#heading=h.qex63c29z2to

I agree with you completely, though. And we should have done what you said to begin with.

With cluster-registry directly imported in the newly updated patch, 'dep ensure' modifies Gopkg.toml which will now contain lines as @gnirodi has pointed out in his comments, in addition to other changes in the same file.